lib/cucumber/core/test/result.rb

```diff
@@ -177,6 +177,24 @@ def ok?(be_strict = false)
           end
         end
 
+        class Ambiguous < Raisable
+          include Result.query_methods :ambiguous
+
+          def describe_to(visitor, *args)
+            visitor.failed(*args)
+            visitor.duration(duration, *args)
+            self
+          end
+
+          def to_s
+            "F"
```

Comment:
Theft? In Open Source? For shame! ;)

Seriously though, yes, that was an initial idea but it somehow ended up starting with the even more basic case of failure rather than a special return type like ambiguous. Given what i have now seen of the code, however, I may be able to make it return the correct type, even if I still can't get the output correct.